packages/quantic/force-app/main/default/lwc/quanticSmartSnippetSuggestions/e2e/fixture.ts

```diff
@@ -0,0 +1,89 @@
+import {SmartSnippetSuggestionsObject} from './pageObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {
+  searchRequestRegex,
+  insightSearchRequestRegex,
+} from '../../../../../../playwright/utils/requests';
+import {InsightSetupObject} from '../../../../../../playwright/page-object/insightSetupObject';
+import {useCaseEnum} from '../../../../../../playwright/utils/useCase';
+import type {QuestionAnswerData} from './data';
+import smartSnippetSuggestionsData from './data';
+
+const pageUrl = 's/quantic-smart-snippet-suggestions';
+
+interface SmartSnippetSuggestionsOptions {}
+
+type QuanticSmartSnippetSuggestionsE2ESearchFixtures = {
+  smartSnippetSuggestionsData: QuestionAnswerData;
```

Comment:
this can just be called `data`, to avoid having very long keys

Reply:
i dont mind it its more precise like this and avoid doing like : `data: data`